src/draw/nema_gfx/lv_draw_nema_gfx_img.c

```diff
@@ -230,7 +244,277 @@ static uint32_t lv_nemagfx_mask_cf_to_nema(lv_color_format_t cf)
         case LV_COLOR_FORMAT_A1:
             return NEMA_A1;
         case LV_COLOR_FORMAT_A2:
-            return NEMA_A2;
+            return NEMA_A2;/**
+ * MIT License
+ *
+ * -----------------------------------------------------------------------------
```

Comment:
It looks like the contents of lv_draw_nema_gfx_fill.c was accidentally pasted into the middle of this file.